python/prairielearn_test.py

```diff
@@ -1,15 +1,66 @@
 # lol
 import sys
 import os
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../lib')))
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../lib"))
+)
```

Comment:
I know this didn't actually change in this PR, but the diff drew my eyes to this. Is this actually necessary? AFAICT, `../../lib` is outside of the PL repo, but the tests still seen to pass just fine. This git-blames to me; I'm inclined to say we can remove this if you want to do an extra bit of cleanup.

Reply:
I think I asked about this somewhere else and had the same question. As far as I can tell it's not needed, will remove.